packages/stateful/actions/actions/nft/makeMintNftAction.tsx

```diff
@@ -149,6 +149,8 @@ export const makeMintNftAction: ActionMaker<MintNftData> = ({ t, address }) => {
       name: '',
       description: '',
       includeExtra: false,
+      audio: undefined,
```

Comment:
Why `undefined` instead of `null`?

Reply:
`undefined` gets removed from objects when they're stringified, and `null` does not. so i generally choose undefined when filling in default empty values :) actions are duplicated and their data fields are stringified into the URL params, so that makes it a bit shorter, for example